doc/qbittorrent-nox.1.md

```diff
@@ -0,0 +1,41 @@
+% QBITTORRENT-NOX(1) Command line Bittorrent client written in C++ / Qt
+% Christophe Dumez <chris@qbittorrent.org>
+% January 16th 2010
+
+# NAME
+qBittorrent-nox - a command line Bittorrent client written in C++ / Qt
+
+
+# SYNOPSIS
+**qbittorrent-nox** `[--d|--daemon] [--webui-port=x] [TORRENT_FILE | URL]...`
+
+**qbittorrent-nox** `--help`
+
+**qbittorrent-nox** `--version`
+
+
+# DESCRIPTION
+qBittorrent-nox is an advanced command-line Bittorrent client written in C++ / Qt
+using the libtorrent-rasterbar library by Arvid Norberg.
```

Comment:
Missing bold text for `qbittorrent-nox` and `libtorrent-rasterbar`.

Reply:
Done.